docs/usage/config-overview.md

```diff
@@ -246,71 +246,3 @@ By having your centralized preset part of each Repository config `extends`, it h
 
 - You still have the ability to change shared settings in a single location
 - Any user viewing the repo can see the preset being extended and trace it back to understand which config is applied
-
-## Mend Renovate App Config
```

Comment:
Can you retain a heading or link here to the new page you created? Assuming that you think a user reading this document might be interested in the app config data which was removed

Reply:
Heading added towards the top of the general configuration overview page.